### ISSUE TITLE
Page for user statistics

### ISSUE THREAD

**Issue by ilhamsyaputra** [stackup-dev/open-source-campaign#23]
**Name of Proposer**
rangerhitam142

**Is your feature request related to a problem? Please describe.**
My feature request does not related to a problem, it's just an improvement to learners and also the StackUp Dev team to get to know which field we as learners are weak or strong in.

**Describe the solution you'd like**
I suggest that StackUp has a special page for user/learner statistics (for each user and all users).
**For specific users**, statistics may contain the percentage of approved or rewarded quests, detailed log of submitted quests, and a summary of quests/fields that the users are strong or weak in (need improvement). (you can also add specific video material for the weak spot, so learners can re-evaluate or re-learning.)
**For all users**, statistics may contain the total number of users who joined the quests, success rate (whether it's approved or rewarded), and also fail rate. These all-user statistics can help the StackUp dev team to focus more on the weak spots.

**Comment by sylo90** [comment]
yup i recommend this suggestion as well ,right now its so difficult to look at the entire summary of our quests
